README.md

```diff
@@ -726,14 +727,15 @@ networks:
 volumes:
   MarkLogic_3n_vol1:
 ```
-2. Use below command to create stack and service:
+2. Use this command to create stack and service:
 ```
 docker stack deploy -c bootstrap-compose.yaml mlstack
 ```
-3. Once the bootstrap host is up and running, enable TLS on App servers(8001, 8002) using the procedure from MarkLogic documentation [https://docs.marklogic.com/guide/security/SSL](https://docs.marklogic.com/guide/security/SSL).
-4. Extract the certificate for bootstrap host and store it in the same directory as compose file. CA certificate/certificate chain used to join the cluster will be stored as Docker secret.
-2. Create files mldb_admin_username.txt and mldb_admin_password.txt to set admin username/password used for joining bootstrap host.
-3. Use below compose file to create node2. Please note MARKLOGIC_JOIN_TLS_ENABLED parameter is set to true and MARKLOGIC_JOIN_CACERT_FILE is set as a Docker secret with it's value set to the CA certificate/certificate chain file path. Please see [Configuration](#Configuration) section for more details on these two parameters.
+3. Once the bootstrap host is up and running, enable HTTPS on App servers(8001, 8002) using the procedure [https://docs.marklogic.com/guide/security/SSL](https://docs.marklogic.com/guide/security/SSL).
```

Comment:
3. Once the bootstrap host is up and running, enable HTTPS on the Admin and Manage app servers (ports 8001 and 8002) using the procedures from the MarkLogic documentation [https://docs.marklogic.com/guide/security/SSL](https://docs.marklogic.com/guide/security/SSL).
4. Extract the certificate for the bootstrap host and store it in the same directory as the compose file. The CA certificate/certificate chain used to join the cluster will be stored as Docker secret.
5. Create files {mldb_admin_username.txt} and {mldb_admin_password.txt} to set the admin username/password used for joining the bootstrap host.
6. Use the compose file below to create node2. Please note the {MARKLOGIC_JOIN_TLS_ENABLED} parameter is set to true and the {MARKLOGIC_JOIN_CACERT_FILE} is set as a Docker secret with the value set to the CA certificate/certificate chain file path. Please see the [Configuration](#Configuration)  section for more details on these two parameters.

Reply:
The step to "Extract the certificate for the bootstrap host" is unclear. Is there documentation on how they should do this?

Reply:
As certificate creation and enabling TLS on Bootstrap is not part of Docker implementation, it is left up to the user to get the certificates used on the Bootstrap host.